site.cfg.example

```diff
@@ -152,6 +153,26 @@
 # include_dirs = /opt/OpenBLAS/include
 # runtime_library_dirs = /opt/OpenBLAS/lib
 
+# OpenBLAS (64-bit ILP64)
+# -----------------------
+# It is possible to also use OpenBLAS compiled with 64-bit integer
+# size (ILP64) but no symbol name changes. To do that, set the
+# environment variables NPY_USE_BLAS_ILP64=1,
+# NPY_BLAS_ILP64_ORDER=openblas_ilp64,
+# NPY_LAPACK_ILP64_ORDER=openblas_ilp64 at build time.
+#
+# Note that mixing both 64-bit and 32-bit BLAS without symbol suffixes
+# in the same application may cause problems due to symbol name
+# clashes, especially with embedded Python interpreters.
+#
+# [openblas_ilp64]
+# libraries = openblas64
```

Comment:
For at least one major package manager (`spack`), the library will actually be called `openblas` without the `64` appended, even for the 64-bit integer variant, which causes this to fail (see my review comment with steps).

I'm assuming that there's really no way we're going to cover the idiosyncrasies of all package managers/build systems here--maybe worth a comment though?!

Reply:
Added notice to site.cfg.example about this.

Reply:
That's right, the 64-bit version gets a different hash in its absolute path when I use spack. Anyway, you've added the comment now, marking as resolved.